docs/index.md

```diff
@@ -5,16 +5,13 @@
 ## 搭建教程
 
 1. 通过 <https://chat.openai.com/chat> 注册并登录。
-2. 打开浏览器开发者工具，切换到 Application 标签页。
-3. 在左侧的 Storage - Cookies 中找到 `__Secure-next-auth.session-token` 一行并复制其值。
-4. 将值填写到 `sessionToken` 字段。
-5. 在左侧的 Storage - Cookies 中找到 `cf_clearance` 一行并复制其值。
-6. 将值填写到 `cloudflareToken` 字段并启用插件。
+2. 安装好插件依赖的服务
+3. 将 puppeteer 插件的**无头模式**关闭
```

Comment:
What if puppeteer used in headless mode ?

Reply:
Because logging in requires turning off headless mode in order to operate.

Reply:
Oh, I thought it would be better to be logged in automatically by the plugin. But it is also not bad as the current workflow.

Reply:
It is hard to implement. Some captcha need manually handle.